application/src/main/java/org/opentripplanner/osm/tagmapping/OsmTagMapper.java

```diff
@@ -59,6 +60,7 @@ public void populateProperties(WayPropertySet props) {
     props.setProperties("mtb:scale=4", noneWayProperties);
     props.setProperties("mtb:scale=5", noneWayProperties);
     props.setProperties("mtb:scale=6", noneWayProperties);
+    props.setProperties("highway=bridleway", withModes(NONE).bicycleSafety(1.3));
```

Comment:
I don't think you need to set the bicycle safety value here since cycling is not allowed?

Reply:
I need to set it because the value will be used in case a bridleway is tagged bicycle=yes.